specs/GetSecurityDescriptorForAppContainerNames/GetSecurityDescriptorForAppContainerNames.md

```diff
@@ -61,6 +61,26 @@ STDAPI GetSecurityDescriptorForAppContainerNames(
 
 If the function succeds, the returned `SECURITY_DESCRIPTOR` must be freed by calling [LocalFree](https://docs.microsoft.com/en-us/windows/desktop/api/winbase/nf-winbase-localfree).
 
+## WinRT
+```c#
+namespace Microsoft.Security.AccessControl
+{
+  struct AppContainerAccess
+  {
+    String appContainerName;
+    UInt32 accessMask;
+  }
+
+  static runtimeclass SecurityDescriptorHelpers
+  {
+    String GetSDDLForAppContainerNames(
```

Comment:
Does this also need a 'static' here? Or is "class level static" sufficient?

Reply:
Good callout. Apparently, I don't author enough MIDL3 to remember the rules here, but you're correct, I do need `static` on the method.

https://docs.microsoft.com/en-us/uwp/midl-3/intro#classes